index.html

```diff
@@ -0,0 +1,1179 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <!-- META -->
+  <meta charset="utf-8" />
+  <meta name="description" content="" />
+  <meta name="Author" content="Codeboxx Wizardz" />
+
+  <title>Rocket Elevators</title>
+  <link rel="icon" href="assets/images/rocketElevators/rocketIcon.png" />
+
+  <!-- MOBILE SETTINGS -->
+  <meta name="viewport" content="width=device-width, maximum-scale=1, initial-scale=1, user-scalable=0" />
+
+  <!-- WEB FONTS -->
+  <link
+    href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600%7CRaleway:300,400,500,600,700%7CLato:300,400,400italic,600,700"
+    rel="stylesheet" type="text/css" />
+
+  <!-- CORE CSS -->
+  <link href="assets/plugins/bootstrap/css/bootstrap.min.css" rel="stylesheet" type="text/css" />
+
+  <!-- REVOLUTION SLIDER -->
+  <link href="assets/plugins/slider.revolution/css/extralayers.css" rel="stylesheet" type="text/css" />
+  <link href="assets/plugins/slider.revolution/css/settings.css" rel="stylesheet" type="text/css" />
+
+  <!-- THEME CSS -->
+  <link href="assets/css/essentials.css" rel="stylesheet" type="text/css" />
+  <link href="assets/css/layout.css" rel="stylesheet" type="text/css" />
+
+  <!-- ROCKET THEME CSS -->
+  <link href="assets/css/rocket-theme.css" rel="stylesheet" type="text/css" />
+
+  <!-- PAGE LEVEL SCRIPTS -->
+  <link href="assets/css/header-1.css" rel="stylesheet" type="text/css" />
+  <link href="assets/css/color_scheme/blue.css" rel="stylesheet" type="text/css" id="color_scheme" />
+  <link rel="icon" type="image/x-icon" href="/assets/RocketElevatorAssets/favicon.ico" />
+
+  <!-- ROBOTO SLAB FONT-->
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab&display=swap" rel="stylesheet" />
+</head>
+
+<body class="smoothscroll enable-animation">
+  <!-- SCROLL TO TOP -->
+  <a href="" id="toTop"></a>
+  <!-- /SCROLL TO TOP -->
+
+  <!-- WRAPPER -->
+  <div id="wrapper">
```

Comment:
This is a comment I'm attaching to this line. I use a comment like this to point out the section I'm talking about.